guider.h

```diff
@@ -165,7 +165,8 @@ class Guider : public wxWindow
     LockPosShiftParams m_lockPosShift;
     bool m_measurementMode;
     double m_minStarHFD;
-    double m_minStarSNR;
+    double m_minAFStarSNR;
+    double m_maxAFStarHFD;

```

Comment:
could we add some comments describing what these variables are? especially to clarify what AF stands for?  I'm finding it a bit confusing to see minHFD without "AF" and maxHFD with "AF" in the name.
Also might be slightly clearer to put the HFD min and max variables together on adjacent lines since they are logically a pair?

Reply:
The two 'AF' methods only control an SNR floor property used in Auto-Find, it is different from the photometric SNR floor used in 'find' operations.  The mis-named class variable was a left-over from my initial cut at this, which anticipated that a max-hfd property would also be used only in Auto-Find.  Obviously, that didn't work very well.